profiles/lib/way_handlers.lua

```diff
@@ -521,7 +521,7 @@ function WayHandlers.blocked_ways(profile,way,result,data)
     local construction = way:get_value_by_key('construction')
 
     -- Of course there are negative tags to handle, too
-    if construction and construction ~= 'no' and construction ~= 'widening' then
+    if construction and construction ~= 'no' and construction ~= 'widening' and construction ~= 'minor' then
```

Comment:
In general this is a good idea in practice osm allows for

> The value of the construction=\* key should be one of those defined bellow and indicates the type of feature being built (you can use any value from from highway=\*, railway=\* and building=\*).

so unless we want to hard-code all of those values I think we're good for now.